dipy/viz/tests/test_ui.py

```diff
@@ -150,11 +150,18 @@ def modify_button_callback(i_ren, obj, button):
     button_test.color = button_color
     # /Button
 
+    # TextBlock
+    text_block_test = ui.TextBlock2D()
+    text_block_test.message = 'TextBlock'
+    text_block_test.color = (0, 0, 0)
+    # /TextBlock
```

Comment:
I just added them for easy readability, since we're putting in all components in a single file. They can be removed, no problem.